README.adoc

```diff
@@ -11,54 +11,55 @@ endif::[]
 https://learnteachcode.org/code-coffee-compendium/home/
 
 == Introduction
-Welcome to the LearnTeachCode Code & Coffee Compendium Github Repository! Maintained by the Baldwin Park & El Sereno/Alhambra study group, this repository hosts the source code for https://learnteachcode.org/code-coffee-compendium/[code-coffee-compendium] resource site. 
+Welcome to the LearnTeachCode Code & Coffee Compendium Github Repository! Maintained by the Code & Coffee Office Hours study group, this repository hosts the source code for https://learnteachcode.org/code-coffee-compendium/[code-coffee-compendium] resource site. 
 
 This site was generated using https://asciidoctor.org/[Asciidoctor] - a static-site generator for the https://asciidoctor.org/docs/asciidoc-syntax-quick-reference[AsciiDoc] markup language.
 
 Want extra coding help? Join our https://learnteachcode.org/slack[Slack group].
 
 == Contributing
-Contributors are welcome to fork this repository and make changes. Please submit a Pull Request with your changes to be reviewed by a repository owner.
+Contributors are welcome to fork this repository and make changes. Please submit a Pull Request (PR) with your changes to be reviewed by a repository owner.
 
 _Not sure where to start?_ There are a number of issues that you can work on. https://github.com/LearnTeachCode/code-coffee-compendium/issues/[Find them here.]
 
 
 == Dependencies
 This project has a number of dependencies that must be met before running on your machine.
 
-1. _Download https://www.ruby-lang.org/en/[Ruby 2.4+]_
-2. _Clone or download this repository to your machine._
-3. _Open a terminal window. Change your current directory to the cloned project directory._
-4. _Install the package bundle by running the following command in the terminal window:_
+1. Docker
+2. Git (for contributors, quicker access to the repository source code)
+3. A Bash terminal (standard terminal environment for *nix and macOS).
 
-----
-gem install bundler:2.1.4
-bundle install
-----
-
-This project also uses PlantUML to generate diagrams. https://plantuml.com/download[Download PlantUML here.]
-
-NOTE: In addition to PlantUML, you will need GraphViz to display the sitemap diagram. https://www.graphviz.org/download/[Download GraphViz here.]
+NOTE: To Windows users: Windows Command Prompt will not satisfy this dependency. Instead, you can use PowerShell or another Bash terminal (like Git Bash for example).
 
 == Viewing the Site Locally
 CAUTION: Please make sure you have all required dependencies before continuing.
 
 1. _Clone or download this repository to your machine._
 2. _Open a terminal window. Change your current directory to the cloned project directory._
-3. _Inside of your terminal, type the following command:_
+3. _Using a Bash terminal, type the following command:_
+   
+`
+sh build.sh
```

Comment:
line isn't rendering as a code block. try 

```
[source]
----
sh build.sh
----
```

as per [callouts](https://gist.github.com/tracphil/87fff6ab2796fd855e62dd852eda502d#callouts) for GFA